patches/net/minecraft/client/gui/Gui.java.patch

```diff
@@ -101,6 +101,23 @@
      }
  
      private void renderCameraOverlays(GuiGraphics p_316735_, DeltaTracker p_348538_) {
+@@ -236,8 +_,14 @@
+                 for (EquipmentSlot equipmentslot : EquipmentSlot.values()) {
+                     ItemStack itemstack = this.minecraft.player.getItemBySlot(equipmentslot);
+                     Equippable equippable = itemstack.get(DataComponents.EQUIPPABLE);
+-                    if (equippable != null && equippable.slot() == equipmentslot && equippable.cameraOverlay().isPresent()) {
+-                        this.renderTextureOverlay(p_316735_, equippable.cameraOverlay().get().withPath(p_380782_ -> "textures/" + p_380782_ + ".png"), 1.0F);
++                    if (equippable != null && equippable.slot() == equipmentslot) {
++                        if (equippable.cameraOverlay().isPresent()) {
++                            this.renderTextureOverlay(p_316735_, equippable.cameraOverlay().get().withPath(p_380782_ -> "textures/" + p_380782_ + ".png"), 1.0F);
++                        }
++
++                        if (equipmentslot == EquipmentSlot.HEAD) {
```

Comment:
Why does it matter whether it's on the head or not? The point is to allow it for any equipment slot if it matches where its supposed to be equipped to.